plugins/c8y_configuration_plugin/src/tests.rs

```diff
@@ -628,6 +628,20 @@ async fn test_handle_config_update_request_tedge_device() -> anyhow::Result<()>
         .with(predicate::always())
         .returning(|_path| false);
 
+    c8y_http_client
+        .expect_download_file()
+        .with(
+            predicate::always(),
+            predicate::always(),
+            predicate::always(),
+            predicate::always(),
+        )
+        .returning(|_, file_name, _, tmp_dir_path| {
+            let downloaded_path = tmp_dir_path.join(file_name);
+            std::fs::File::create(&downloaded_path)?;
+            Ok(downloaded_path)
+        });
+
     // Mock download endpoint for the plugin to download config file update from the cloud
     let config_update_cloud_url_path = "/some/cloud/url";
     let _download_config_url_mock = mockito::mock("GET", config_update_cloud_url_path)
```

Comment:
Now that the download call is going to the `MockC8YHttpProxy`, this mockito mock of the server is not needed as we don't even hit this server anymore(bypassed with the proxy mock).

Reply:
True, after removing there is no problem.